### PR TITLE
Add Creative Commons to Documentation files

### PR DESCRIPTION
**Pull request by peterschwarz**

### CODE REVIEW

**Comment by peterschwarz** [comment]
@dcmiddle Added a LICENSE file to the `docs/source/images` directory.

File: docs/source/app_developers_guide/ubuntu.rst [hunk 1]
```diff
@@ -505,12 +505,5 @@ The output of the command will be similar to this:
...
 
-
-Stopping Sawtooth Components
-============================
-
-To stop the validator, enter CTRL-c several times in the validator's terminal
-window.
-
-Stop the REST API and transaction processors by entering a single CTRL-c in
-the appropriate windows.
```
Comment (dcmiddle) [inline docs/source/app_developers_guide/ubuntu.rst]: Did you intend to delete this block?

Comment (peterschwarz) [reply docs/source/app_developers_guide/ubuntu.rst]: Looks like a mistake on my part after a rebase - will fix

**Comment by peterschwarz** [comment]
The SVG files were most likely generated using the same tool that produced the PDF artifacts.  All the images are covered by the LICENSE file in the `docs/source/images` directory.

I may have missed removing the CC 4.0 license from the file you mentioned.

Yeah, the recommendation was to put that info at the bottom.

**Comment by peterschwarz** [comment]
Updated to remove remaining SVG header

**Pull request merged**